### PR TITLE
F#343 hand finder

### PR DESCRIPTION
**Pull request by VahidAminZ**

### CODE REVIEW

**Comment by VahidAminZ** [comment]
fixes #343

File: sr_utilities/scripts/sr_utilities/hand_finder.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        ros_pack = rospkg.RosPack()
+        ethercat_path = ros_pack.get_path('sr_ethercat_hand_config')
+        self.friction_compensation = {}
+        self.host_control = {}
+        self.motor_control = {}
+        for hand in mapping:
+            self.friction_compensation[hand] = ethercat_path + '/controls/' \
+                + 'friction_compensation.yaml'
+            host_path = ethercat_path + '/controls/host/' + mapping[hand] + '/'
+            self.host_control[hand] = \
```
Comment (toliver) [inline sr_utilities/scripts/sr_utilities/hand_finder.py]: All these dictionaries will have the serial number of the hand as the key.

Isn't this a bit strange from the user point of view? It seems better to use `mapping[hand]` (i.e. rh or lh) instead.

File: sr_utilities/scripts/sr_utilities/hand_finder.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        self.mapping = mapping
+        self.joint_prefix = joint_prefix
+
+
+class HandJoints(object):
+    def __init__(self, mapping):
+        """
+
+        """
+        joints = ['ffj0', 'ffj3', 'ffj4', 'mfj0', 'mfj3', 'mfj4', 'rfj0',
```
Comment (toliver) [inline sr_utilities/scripts/sr_utilities/hand_finder.py]: Upper case maybe?

File: sr_utilities/scripts/sr_utilities/hand_finder.py (hunk 1 above)
Comment (VahidAminZ) [inline sr_utilities/scripts/sr_utilities/hand_finder.py]: changed them accordingly

**Pull request merged**